bot/cogs/utilities.py

```diff
@@ -193,7 +193,7 @@ async def drink(self, ctx: Context) -> None:
 
             await ctx.send(msg)
 
-    @command(brief="Shows the latency between Friendo and the Discord API")
+    @command(brief="Shows the latency between Friendo and the disnake API")
```

Comment:
It's still the Discord API.
```suggestion
    @command(brief="Shows the latency between Friendo and the Discord API")
```